<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PracticeClass</class>
 <widget class="QMainWindow" name="PracticeClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>489</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>viewImg</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{background-color:rgb(255, 250, 249)}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>406</width>
         <height>410</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="labDisplayPix">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>178</width>
     <height>340</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>524287</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>dockWidget</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTreeWidget" name="treeFiles">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>300</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="headerVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="headerCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="headerMinimumSectionSize">
        <number>120</number>
       </attribute>
       <attribute name="headerDefaultSectionSize">
        <number>120</number>
       </attribute>
       <attribute name="headerHighlightSections">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>item</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>itemType</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonFollowStyle</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actAddFolder"/>
   <addaction name="actAddFiles"/>
   <addaction name="actDeleteItem"/>
   <addaction name="actTraversalItems"/>
   <addaction name="separator"/>
   <addaction name="actZoomIn"/>
   <addaction name="actZoomOut"/>
   <addaction name="actZoomRealSize"/>
   <addaction name="actZoomFitH"/>
   <addaction name="actZoomFitW"/>
   <addaction name="actDockFloating"/>
   <addaction name="actDockVisible"/>
   <addaction name="actQuit"/>
  </widget>
  <action name="actAddFolder">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="viewImg.qrc">
     <normaloff>:/pics/vorctorIcon/文件夹.png</normaloff>:/pics/vorctorIcon/文件夹.png</iconset>
   </property>
   <property name="text">
    <string>添加目录</string>
   </property>
   <property name="toolTip">
    <string>添加目录</string>
   </property>
  </action>
  <action name="actAddFiles">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="viewImg.qrc">
     <normaloff>:/pics/vorctorIcon/mediaAdd.ico</normaloff>:/pics/vorctorIcon/mediaAdd.ico</iconset>
   </property>
   <property name="text">
    <string>添加文件</string>
   </property>
   <property name="toolTip">
    <string>添加文件</string>
   </property>
  </action>
  <action name="actDeleteItem">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="viewImg.qrc">
     <normaloff>:/pics/vorctorIcon/删除筛选项.png</normaloff>:/pics/vorctorIcon/删除筛选项.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>删除节点</string>
   </property>
  </action>
  <action name="actTraversalItems">
   <property name="icon">
    <iconset>
     <normaloff>vorctorIcon/目录遍历.png</normaloff>vorctorIcon/目录遍历.png</iconset>
   </property>
   <property name="text">
    <string>遍历</string>
   </property>
   <property name="toolTip">
    <string>遍历节点</string>
   </property>
  </action>
  <action name="actZoomIn">
   <property name="icon">
    <iconset resource="viewImg.qrc">
     <normaloff>:/pics/vorctorIcon/zoomIn.ico</normaloff>:/pics/vorctorIcon/zoomIn.ico</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="iconText">
    <string>放大</string>
   </property>
   <property name="toolTip">
    <string>放大</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actZoomOut">
   <property name="icon">
    <iconset resource="viewImg.qrc">
     <normaloff>:/pics/vorctorIcon/zoomOut.ico</normaloff>:/pics/vorctorIcon/zoomOut.ico</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string>缩小</string>
   </property>
  </action>
  <action name="actZoomRealSize">
   <property name="icon">
    <iconset resource="viewImg.qrc">
     <normaloff>:/pics/vorctorIcon/rotateLeft.ico</normaloff>:/pics/vorctorIcon/rotateLeft.ico</iconset>
   </property>
   <property name="text">
    <string>真实大小</string>
   </property>
   <property name="toolTip">
    <string>真实大小</string>
   </property>
  </action>
  <action name="actZoomFitH">
   <property name="icon">
    <iconset resource="viewImg.qrc">
     <normaloff>:/pics/vorctorIcon/高度.png</normaloff>:/pics/vorctorIcon/高度.png</iconset>
   </property>
   <property name="text">
    <string>适合高度</string>
   </property>
   <property name="toolTip">
    <string>适合高度</string>
   </property>
  </action>
  <action name="actZoomFitW">
   <property name="icon">
    <iconset resource="viewImg.qrc">
     <normaloff>:/pics/vorctorIcon/宽度.png</normaloff>:/pics/vorctorIcon/宽度.png</iconset>
   </property>
   <property name="text">
    <string>适合宽度</string>
   </property>
   <property name="toolTip">
    <string>适合宽度</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="viewImg.qrc">
     <normaloff>:/pics/vorctorIcon/退出.png</normaloff>:/pics/vorctorIcon/退出.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actDockFloating">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="viewImg.qrc">
     <normaloff>:/pics/vorctorIcon/浮动.png</normaloff>:/pics/vorctorIcon/浮动.png</iconset>
   </property>
   <property name="text">
    <string>浮动</string>
   </property>
   <property name="toolTip">
    <string>窗口悬浮</string>
   </property>
  </action>
  <action name="actDockVisible">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="viewImg.qrc">
     <normaloff>:/pics/vorctorIcon/可见.png</normaloff>:/pics/vorctorIcon/可见.png</iconset>
   </property>
   <property name="text">
    <string>可见</string>
   </property>
   <property name="toolTip">
    <string>窗口可见</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="viewImg.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actAddFolder</sender>
   <signal>triggered()</signal>
   <receiver>PracticeClass</receiver>
   <slot>onActAddFolderTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actAddFiles</sender>
   <signal>triggered()</signal>
   <receiver>PracticeClass</receiver>
   <slot>onActAddFilesTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeFiles</sender>
   <signal>currentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</signal>
   <receiver>PracticeClass</receiver>
   <slot>onTreeFilesCurrentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeFiles</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>PracticeClass</receiver>
   <slot>onTreeFilesCustomContextMenuRequired(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actDeleteItem</sender>
   <signal>triggered()</signal>
   <receiver>PracticeClass</receiver>
   <slot>onActDeleteItemTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actTraversalItems</sender>
   <signal>triggered()</signal>
   <receiver>PracticeClass</receiver>
   <slot>onActTraversalTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actZoomIn</sender>
   <signal>triggered()</signal>
   <receiver>PracticeClass</receiver>
   <slot>onActZoomInTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actZoomOut</sender>
   <signal>triggered()</signal>
   <receiver>PracticeClass</receiver>
   <slot>onActZoomOutTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actZoomRealSize</sender>
   <signal>triggered()</signal>
   <receiver>PracticeClass</receiver>
   <slot>onActZoomRealSizeTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actZoomFitW</sender>
   <signal>triggered()</signal>
   <receiver>PracticeClass</receiver>
   <slot>onActZoomFitWTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actZoomFitH</sender>
   <signal>triggered()</signal>
   <receiver>PracticeClass</receiver>
   <slot>onActZoomFitHTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockWidget</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>PracticeClass</receiver>
   <slot>onDockWidgetVisibilityChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actDockVisible</sender>
   <signal>toggled(bool)</signal>
   <receiver>PracticeClass</receiver>
   <slot>onActDockVisibleToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actDockFloating</sender>
   <signal>triggered(bool)</signal>
   <receiver>PracticeClass</receiver>
   <slot>onActDockFloatTriggered(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockWidget</sender>
   <signal>topLevelChanged(bool)</signal>
   <receiver>PracticeClass</receiver>
   <slot>onDockTopLevelChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>PracticeClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onActAddFolderTriggered()</slot>
  <slot>onActAddFilesTriggered()</slot>
  <slot>onTreeFilesCurrentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
  <slot>onTreeFilesCustomContextMenuRequired(QPoint)</slot>
  <slot>onActDeleteItemTriggered()</slot>
  <slot>onActTraversalTriggered()</slot>
  <slot>onActZoomInTriggered()</slot>
  <slot>onActZoomOutTriggered()</slot>
  <slot>onActZoomRealSizeTriggered()</slot>
  <slot>onActZoomFitWTriggered()</slot>
  <slot>onActZoomFitHTriggered()</slot>
  <slot>onActDockVisibleToggled(bool)</slot>
  <slot>onDockWidgetVisibilityChanged(bool)</slot>
  <slot>onActDockFloatTriggered(bool)</slot>
  <slot>onDockTopLevelChanged(bool)</slot>
 </slots>
</ui>
